### PR TITLE
Unifying modules into one project

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by zacanbrcom** [comment]
is a leftover that? https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/420261687679d8891438c45ad1f19a7acc10675e/server/README.md

**Comment by asatklichov** [comment]
Three modules removed and parent folder for java is changed to "server". Tested locally and on Che application working fine. All tests are passed, sonar-cloud test reports are fine.

**Comment by asatklichov** [comment]
> is a leftover that? https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/420261687679d8891438c45ad1f19a7acc10675e/server/README.md

It was take from one removed module, was empty with only name of project.  We can add there some useful information there.

**Comment by zacanbrcom** [comment]
> > is a leftover that? https://github.com/eclipse/che-che4z-lsp-for-cobol/blob/420261687679d8891438c45ad1f19a7acc10675e/server/README.md
> 
> It was take from one removed module, was empty with only name of project. We can add there some useful information there.

I propose to remove it and maybe introduce it later if needed.

File: server/src/test/java/com/ca/lsp/cobol/positive/PositiveTest.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ void test(CobolText text) {
     if (text == null) {
       return;
     }
-    log.debug("Processing: " + text.getFileName());
+      LOG.debug("Processing: " + text.getFileName());
```
Comment (zacanbrcom) [inline server/src/test/java/com/ca/lsp/cobol/positive/PositiveTest.java:63]: looks like there is an extra space

File: server/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -23,7 +23,8 @@
 			</pattern>
 		</encoder>
 	</appender>
-	<root level="warn">
+<!--	<root level="warn">-->
```
Comment (zacanbrcom) [inline server/src/test/resources/logback-test.xml:26]: better remove it if no longer necessary ;)

**Comment by temanbrcom** [comment]
I'll squash on merge, so this email issue will disappear

**Pull request merged**